src/SFA.DAS.FAT.Web.UnitTests/Controllers/CoursesControllerTests/WhenGettingCourseProviderDetails.cs

```diff
@@ -65,36 +65,28 @@ public class WhenGettingCourseProviderDetails
             Assert.IsNotNull(actualModel.AdditionalCourses);
             Assert.IsNotNull(actualModel.AdditionalCourses.Courses);
             actualModel.Location.Should().BeNullOrEmpty();
-            cookieStorageService.Verify(x=>x.Update(Constants.LocationCookieName,It.IsAny<LocationCookieItem>(), It.IsAny<int>()), Times.Never);
-            
+            cookieStorageService.Verify(x => x.Update(Constants.LocationCookieName, It.IsAny<LocationCookieItem>(), It.IsAny<int>()), Times.Never);
+
         }
 
-        [Test]
-        [MoqInlineAutoData("2023-Mar-31", 2018,2019)]

```

Comment:
@DevenKShah why did you not change the **expectedDateFrom** and **expectedDateTo**  values but instead made an inline literal code change?